Formula/authoscope.rb

```diff
@@ -15,14 +15,17 @@ class Authoscope < Formula
   end
 
   depends_on "rust" => :build
-  depends_on "openssl@1.1"
 
   uses_from_macos "zlib"
 
+  on_linux do
+    depends_on "openssl@3" # Uses Secure Transport on macOS
```

Comment:
I don't think this uses OpenSSL on macOS:
```console
❯ otool -L authoscope/0.8.1/bin/authoscope
authoscope/0.8.1/bin/authoscope:
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1311.100.3)
	/System/Library/Frameworks/Security.framework/Versions/A/Security (compatibility version 1.0.0, current version 60158.100.133)
	/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (compatibility version 150.0.0, current version 1858.112.0)
	/usr/lib/libz.1.dylib (compatibility version 1.0.0, current version 1.2.11)
	/usr/lib/libiconv.2.dylib (compatibility version 7.0.0, current version 7.0.0)
	/usr/lib/libresolv.9.dylib (compatibility version 1.0.0, current version 1.0.0)
```

I think OpenSSL on Linux is due to [`native-tls`](https://github.com/sfackler/rust-native-tls#rust-native-tls)
> Specifically, this crate uses SChannel on Windows (via the [schannel](https://crates.io/crates/schannel) crate), Secure Transport on macOS (via the [security-framework](https://crates.io/crates/security-framework) crate), and OpenSSL (via the [openssl](https://crates.io/crates/openssl) crate) on all other platforms.